### PR TITLE
Added error logs in the healthcheck code of zrepl pool watcher

### PR DESCRIPTION
**Pull request by Moteesh**
added error logs in CheckForZreplInitial and CheckForZreplContinuous methods to print the error message upon failure.

Signed-off-by: moteesh <moteesh@gmail.com>

### CODE REVIEW

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 1]
```diff
@@ -183,6 +184,7 @@ func CheckForZreplContinuous(ZreplRetryInterval time.Duration) {
 			time.Sleep(ZreplRetryInterval)
 			continue
 		}
+		glog.Errorf("Error in CheckForZreplContinuous : %v", err)
```
[the remark this answers is not in the document]
Comment (Moteesh) [reply cmd/cstor-pool-mgmt/pool/pool.go]: have done the requested changes.

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 2]
```diff
@@ -171,6 +171,7 @@ func CheckForZreplInitial(ZreplRetryInterval time.Duration) {
 			glog.Infof("Waiting for zpool replication container to start...")
 			continue
 		}
+		glog.Errorf("Error in CheckForZreplInitial : %v", err)
```
Comment (avishnu) [inline cmd/cstor-pool-mgmt/pool/pool.go]: Won't err print nil here?

File: cmd/cstor-pool-mgmt/pool/pool.go (hunk 1 above)
Comment (avishnu) [inline cmd/cstor-pool-mgmt/pool/pool.go]: Won't err print nil here?

Comment (Moteesh) [reply cmd/cstor-pool-mgmt/pool/pool.go]: done

Comment (avishnu) [inline cmd/cstor-pool-mgmt/pool/pool.go]: Please re-phrase the log message to something like "zpool status returned error in CheckForZreplContinuous: %v"

**Pull request merged**